eng/DownloadSharedSource.ps1

```diff
@@ -1,11 +1,17 @@
+#Requires -Version 6.0
+$ErrorActionPreference = 'Stop'
+
+$currentProgressPreference = $ProgressPreference
+$ProgressPreference = 'SilentlyContinue'
+
 $downloadPath = Resolve-Path (Join-Path $PSScriptRoot '..' 'src' 'assets' 'Azure.Core.Shared')
-$files = "ClientDiagnostics.cs", "ArrayBufferWriter.cs", "DiagnosticScope.cs"
-$baseUrl = "https://raw.githubusercontent.com/Azure/azure-sdk-for-net/master/sdk/core/Azure.Core/src/Shared/"
+$files = 'ClientDiagnostics.cs', 'ArrayBufferWriter.cs', 'DiagnosticScope.cs'
+$baseUrl = 'https://raw.githubusercontent.com/Azure/azure-sdk-for-net/master/sdk/core/Azure.Core/src/Shared/'
 
 foreach ($file in $files)
 {
     $text = Invoke-WebRequest -Uri "$baseUrl/$file";
-    $text.Content.Replace("#nullable enable", "#pragma warning disable CS8600, CS8604, CS8605").Trim() | Out-File "$downloadPath/$file"
+    $text.Content.Replace('#nullable enable', '#pragma warning disable CS8600, CS8604, CS8605').Trim() | Out-File (Join-Path $downloadPath $file)
 }
 
-exit 0
\ No newline at end of file
+$ProgressPreference = $currentProgressPreference
```

Comment:
I don't think you need to restore this value anymore.

Reply:
Removed restoring it.